### PR TITLE
Fix common function

### PR DESCRIPTION
**Pull request by dtrunov**
Fix problem in get_data_policy_sql, implement method for sleep in common function

### CODE REVIEW

**Comment by dtrunov** [comment]
@VVeremjova please review

File: user_modules/shared_testcases/commonFunctions.lua [hunk 1]
```diff
@@ -745,14 +746,25 @@ end
 ---------------------------------------------------------------------------------------------
 function commonFunctions:SDLForceStop(self)
   os.execute("ps aux | grep smart | awk \'{print $2}\' | xargs kill -9")
-  os.execute("sleep 1")
+  sleep(1)
+end
+
+function sleep(n)
```
Comment (VVeremjova) [inline user_modules/shared_testcases/commonFunctions.lua]: @dtrunov Change function from global to commonFunctions:sleep(n)

File: user_modules/shared_testcases/commonFunctions.lua [hunk 2]
```diff
@@ -745,14 +746,25 @@ end
...
 end
 
 function check_file_existing(path)
   local file = io.open(path, "r")
   if file == nil then
     print("File doesnt exist, path:"..path)
     assert(false)
+  else
+    local ok, err, code = file:read(1)
+    if code == 21 then
```
Comment (VVeremjova) [inline user_modules/shared_testcases/commonFunctions.lua]: @dtrunov magic number. Add either comment or some variable for 21

File: user_modules/shared_testcases/commonFunctions.lua (hunk 1 above)
Comment (dtrunov) [inline user_modules/shared_testcases/commonFunctions.lua]: @VVeremjova Fix within 23b153e

File: user_modules/shared_testcases/commonFunctions.lua (hunk 2 above)
Comment (dtrunov) [inline user_modules/shared_testcases/commonFunctions.lua]: @VVeremjova Fix within 23b153e

**Comment by dtrunov** [comment]
@VVeremjova Please review lates changes

**Comment by dtrunov** [comment]
@LuxoftAKutsan Please merge this PR

**Pull request merged**